src/components/GraphiQL.js

```diff
@@ -250,6 +251,11 @@ export class GraphiQL extends React.Component {
           title="Prettify Query (Shift-Ctrl-P)"
           label="Prettify"
         />
+        <ToolbarButton
+          onClick={this.handleMergeQuery}
+          title="Merge Query (Shift-Ctrl-M)"
+          label="Merge"
```

Comment:
Eh, I'm leaning more towards not having this as a button... I understand a button would help to show this feature exists, but:
- We need to consider the space at the top bar; it's not too bad right now, but I'd like us to think really carefully about which features to put in the top bar - it's gotta be a feature that sees the major usage overall.
- I don't know what to call this feature :D I definitely don't think "Merge" is a good title for this feature/button since it's "Merge fragments into queries", which is then too long. 

Could we keep this as a shortcut-only feature for now?